### PR TITLE
infra: fix Python3 support - iteritems

### PR DESCRIPTION
**Pull request by mnecas**
Backport of https://github.com/oVirt/ovirt-ansible-infra/pull/81

### CODE REVIEW

**Comment by mnecas** [comment]
ci test please

File: roles/infra/roles/aaa_jdbc/tasks/main.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@
     - users
 
 - name: Update users according to attributes
-  command: "{{ aaa_jdbc_prefix }}/ovirt-aaa-jdbc-tool user edit {{ item.name }} {% for attr, value in item['attributes'].iteritems() %} --attribute={{ attr }}='{{ value }}' {% endfor %}"
+  command: "{{ aaa_jdbc_prefix }}/ovirt-aaa-jdbc-tool user edit {{ item.name }} {% for attr, value in item['attributes'].items() %} --attribute={{ attr }}='{{ value }}' {% endfor %}"
```
Comment (mwperina) [inline roles/infra/roles/aaa_jdbc/tasks/main.yml:37]: According to https://www.python.org/dev/peps/pep-0469/ shouldn't we rather use `iter(item['attributes'].items())` ?
Marcin, what do you think?

@tinez

Comment (tinez) [reply roles/infra/roles/aaa_jdbc/tasks/main.yml:37]: It doesn't make that much of a difference here. First of all, we're within an iteration context and not assigning to a variable. `.items()` gives us a list in py2 and py3 gives a view. Even if we use the extra `iter()` layer of wrapping, we've already created the whole list object in memory, so we're missing the point here.

Unless we expect the `item['attributes']` dictionary to be really big LGTM.

Comment (mwperina) [reply roles/infra/roles/aaa_jdbc/tasks/main.yml:37]: OK, thanks

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**